bvm/ballerina-core/src/main/resources/MessagesBundle.properties

```diff
@@ -142,3 +142,7 @@ invalid.action.first.parameter = first parameter of the action should be of conn
 retry.stmt.not.allowed.here = retry statement not allowed here
 invalid.retry.count = invalid retry count
 arguments.count.mismatch = arguments count mismatch: expected {0}, found {1}
+no.such.documentable.attribute = no such documentable attribute {0} in ''{1}''
```

Comment:
These properties are no longer used for semantic error. Please remove them.

Reply:
+1